### ISSUE TITLE
After Merge mod wont load

### ISSUE THREAD

**Issue by coolmoo** [LazyGuyWithRSI/grim_dawn_mod_merger#1]
After the merging happens i get a folder called what i named it to be but when i select it in the game i dont get any mods loaded what am i doing wrong

**Comment by SP77** [comment]
Same here. Merged Dawn of Masteries and Smash N Grab without issue (at least none came up), but when I loaded them into the game, DoM wasn't active. Any help would be appreciated. Thanks for your work. Enjoy the new year.

**Comment by zrovihr** [comment]
same here with DoM + Smash N Grab. both doesnt work

**Comment by DanHaltsev** [comment]
And same here as well. Trying merge Run Speed + Smash N Grab and none of it works. LazyGuyWithRSI, are you alive?

**Comment by LazyGuyWithRSI** [comment]
Yes I alive. Sadly I've had no time to maintain, I think I'll update the readme. I was mostly hoping people would see it is possible to build mods from the command line instead of the GUI tools. I've started from scratch on a new merger, will link to it from this repo when it's done.

**Comment by pylon404** [comment]
I noticed this same issue and looked into the files it makes. It seems like it properly grabs everything from the mods selected, but it doesn't create the required .arz file in the database folder like the component mods have. The solution I found is to:
1. Use the Asset Manager to extract the game assets, specifically the templates (if you have troubles, move database.arz, templates.arc, ArchiveTool.exe, and zlibwapi.dll into a separate folder and run archivetool from command line to get the extracted assets) and put the extracted templates folder (in my case) in the working directory of the Asset Manager (so putting the templates folder in Grim Dawn/database next to templates.arc). Guides I found on modding with Asset Manager weren't super clear on this fact of putting the extracted files into the working directory but it makes sense and fixed my errors of it not finding the templates. 
2. Go to Mod > Select > (name of your combined mod) in Asset Manager and then use Archive > Build and it should spit out a list of finding a bunch of templates that I literally just put in. 
3. Again in Asset Manager use Build > Build and wait for it to finish.
4. Check the file size of the new combined database vs the databases you used to make the combo (I'm guessing that in all cases it should be larger than individual files you started with) 
5. If all worked good then open GD and run in the combo mod's custom world. 

In my case I combined Smash N Grab, Shattered Affixes, and Extended Character Progression all off of Nexus Mods. I just got it working so I will obviously play it more to make sure everything works but the crafting recipes and expanded inventory from both Smash N Grab and Shattered Affixes are there. Hope this helps to anyone who had the same kind of issue!
